package.json

```diff
@@ -10,42 +10,52 @@
     "src"
   ],
   "scripts": {
-    "build": "npm run test && npm run clean && npm run lint-js && npm run build-es5 && npm run build-webpack",
-    "build-doc": "jsdoc  --package package.json -d doc ./src/**/*.js",
-    "build-es5": "babel src --out-dir lib ",
-    "build-webpack": "webpack -p --progress --colors --display-error-details",
+    "babel": "babel src --out-dir lib --ignore '**/test/*'",
+    "build": "yarn run babel && yarn run webpack",
     "clean": "rm -rf dist lib doc",
     "dev": "webpack-dev-server --config webpack.config.dev.js",
-    "lint-js": "eslint 'src/**/*.js' --fix",
-    "test": "mocha --compilers js:babel-core/register --recursive"
+    "docs": "jsdoc  --package package.json -d doc ./src/**/*.js",
+    "lint": "eslint './src/**/*.js'",
+    "prepublish": "yarn run build",
+    "prebuild": "yarn run clean && yarn run lint && yarn run test",
+    "test": "mocha --recursive --compilers js:babel-core/register test/*.js",
+    "webpack": "webpack -p --progress --colors --display-error-details"
   },
   "dependencies": {
     "babel-runtime": "^6.26.0",
     "jsonrpc-dispatch": "^2.3.0",
     "mutation-observer": "^1.0.3"
   },
   "devDependencies": {
-    "babel-cli": "^6.7.5",
-    "babel-core": "^6.7.7",
-    "babel-loader": "^6.2.4",
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.3",
+    "babel-eslint": "10.0.1",
+    "babel-loader": "^7.1.5",
     "babel-plugin-array-includes": "^2.0.3",
-    "babel-plugin-transform-object-assign": "^6.22.0",
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-preset-env": "^1.6.0",
-    "chai": "^3.5.0",
-    "eslint": "^2.13.1",
-    "eslint-config-airbnb": "^9.0.1",
-    "eslint-plugin-import": "^1.10.2",
-    "eslint-plugin-jsx-a11y": "^1.5.3",
-    "eslint-plugin-react": "^5.2.2",
-    "jsdoc": "^3.4.0",
-    "jsdom": "9.6.0",
-    "jsdom-global": "2.1.0",
-    "matchdep": "^1.0.1",
-    "mocha": "^3.1.0",
-    "sinon": "^1.17.6",
-    "webpack": "^2.5.1",
-    "webpack-dev-server": "^2.4.5"
+    "babel-polyfill": "^6.26.0",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-react": "^6.24.1",
+    "chai": "^4.2.0",
+    "eslint": "^5.9.0",
```

Comment:
I would prefer if we aligned with our sharable eslint configuration: https://github.com/cerner/eslint-config-terra/blob/master/package.json